thorlcr/graph/thgraphslave.cpp

```diff
@@ -1151,6 +1177,11 @@ bool CJobSlave::getWorkUnitValueBool(const char *prop, bool defVal) const
     return workUnitInfo->queryPropTree("Debug")->getPropBool(propName.toLowerCase().str(), defVal);
 }
 
+void CJobSlave::debugRequest(CMessageBuffer &msg, const char *request) const
+{
+    watchdog->debugRequest(msg, request);
```

Comment:
wathdog can be disabled, should do a if (watchdog) here and anywhere else similar to this.